dg/media/coco/app/scripts/views/form.js

```diff
@@ -237,6 +236,20 @@ define([
         render_labels: function(){
             $f_el = this.$("#form_template_render");
             $f_el.append(this.form_template(this.labels));
+
+            var partner_name = User.get('partner_name');
+            // if (partner_name != "umed"){
+            if (partner_name != all_configs.misc.tweak_fields_based_on_partner[0]){
```

Comment:
Shouldn't we read partner names from a list, in case tomorrow we have a multiple partners asking for the same tweak.